src/app/GUI/Dialogs/scenesettingsdialog.cpp

```diff
@@ -153,9 +158,20 @@ QString SceneSettingsDialog::getCanvasName() const {
 }
 
 FrameRange SceneSettingsDialog::getFrameRange() const {
```

Comment:
Please make the code style consistent in SceneSettingsDialog::getFrameRange() with existing Enve code(e.g., clean up indentations)